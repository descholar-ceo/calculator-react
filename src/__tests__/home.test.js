import renderer from 'react-test-renderer';
import home from '../pages/home';

it('Render homepage correctly', () => {
  const homePage = renderer.create(<home />).toJSON();

  expect(homePage).toMatchSnapshot();
});
