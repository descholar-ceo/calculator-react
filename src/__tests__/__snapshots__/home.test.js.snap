// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render homepage correctly 1`] = `<home />`;
